cyclops/query/ops.py

```diff
@@ -1696,6 +1696,40 @@ def __call__(self, table: TableTypes) -> Subquery:
         return select(table).where(cond).subquery()
 
 
+@dataclass
+class Apply:
+    """Apply a function to column(s).
+
+    The function must take a sqlalchemy column object and also return a column object.
```

Comment:
Should this perhaps be encoded in the `Callable[[arg_type_1, ...., arg_type_n], return_type]` format below?

Reply:
Added now